src/pages/DataAccessRequestApplication_new.js

```diff
@@ -190,6 +192,47 @@ const DataAccessRequestApplicationNew = (props) => {
     });
   }, [datasets]);
 
+  const resetForcedScrollDebounce = useCallback(() => {
```

Comment:
Once validation is triggered, this method is called every time the user scrolls the window. As a consequence of that, the `init` callback is also called which triggers infinite reloading when the user pages up and down the form.

Reply:
good catch - i will investigate this, thank you

Reply:
this should be fixed!

Reply:
I'm still seeing a similar behavior after a fresh pull and build. After trying to submit an incomplete form, the validation is triggered. Scrolling anywhere on the page triggers all of the API calls that populate the page content. Here's a screen-cap of the requests made after scrolling the page once. Let's follow up in person so I can show you a repeatable case.

<img width="1398" alt="Screenshot 2023-03-15 at 6 22 25 AM" src="https://user-images.githubusercontent.com/116679/225280705-81c83428-357f-4766-9381-87f6592a7a6b.png">

Reply:
I figured it out! No idea why my changes made this worse, but basically because: `init` depends on `onScroll` which depends on `forcedScroll` - which controls debounce on forcing scroll, to ensure scroll doesn't occur too quickly. since submitting scrolls to one of the errors, it triggers a debounce update / sets debounce timer. every time the debounce timer function is updated (see: often), then it would trigger onScroll update... then init update... downloading everything again. removing the debounce mechanism seems to make the scroll more stable and does not trigger init update anymore.